compiler/include/driver.h

```diff
@@ -286,4 +286,6 @@ namespace chpl {
 
 extern chpl::Context* gContext;
 
+extern std::vector<std::pair<std::string,std::string>> gDynoParams;
```

Comment:
let's leave it out of the header

Reply:
It is used in `config.cpp` too, that's where it gets populated with name/value pairs